doc/source/tune/api/logging.rst

```diff
@@ -59,8 +59,10 @@ See the :doc:`tutorial here </tune/examples/tune-mlflow>`.
 
 .. autosummary::
     :nosignatures:
+    :toctree: doc/
 
-    tune.logger.mlflow.MLflowLoggerCallback
+    ~air.integrations.mlflow.MLflowLoggerCallback
+    ~air.integrations.mlflow.setup_mlflow
```

Comment:
nit: Should we include the setup funcs for the others as well?

Reply:
IMO if something is part of the public API, there should be user-facing docs about it. I'll add the other setup funcs now.

Reply:
Turns out the only missing one was `wandb` - there's no `setup_comet`, etc. But anyway, it's been added.

Reply:
Not sure I follow this comment entirely, but as of right now this information is being captured in [this](https://docs.ray.io/en/master/tune/examples/tune-wandb.html#tune-wandb-api-reference) "examples" page.